src/iop/colorchecker.c

```diff
@@ -1210,6 +1211,16 @@ static gboolean checker_button_press(GtkWidget *widget, GdkEventButton *event,
     self->gui_update(self);
     return TRUE;
   }
+  else if((event->button == 1) &&
+          ((event->state & GDK_MOD1_MASK) == GDK_MOD1_MASK) &&
+          (self->request_color_pick == DT_REQUEST_COLORPICK_MODULE))
+  {
+    // Make the picked color match target color on its chroma components
+    // NB: we don't match the L channel because the user is still left with
+    //     the option to increase the exposure by himself/herself
```

Comment:
I'm wondering if this is the right option. Maybe we should also match the L channel to have the correct color and let user adjust if necessary? It seems easier this way than letting the user try to adjust the L channel to match the color.

Reply:
My take on this is that the L* values are only going to match the reference values in very specific cases during the shot and there are probably more risks at trying to adjust L* values differently depending on their chromaticity.

For example, if the target is not lit homogeneously (close speedlight w/ rapid falloff, or target hold w/ an angle to light source), matching L* would mess up L* values for the different patches depending on their physical position in the scene.

Hence my comment about letting users adjust L* globally on the image.

But I have no strong opinion on this. What's yours ?

Reply:
> For example, if the target is not lit homogeneously

If so, the whole colorchecker profiling becomes useless. I think you have to assume in the code that the lightingt has been done properly. That's user's responsibility.